hydroshare/urls.py

```diff
@@ -131,7 +134,7 @@
     # one out.
 
     # url("^$", direct_to_template, {"template": "index.html"}, name="home"),
-    url(r"^tests/$", direct_to_template, {"template": "tests.html"}, name="tests"),
+    # QUNIT_TESTS_OFF
 
```

Comment:
what is the reason for this change?

Reply:
good catch. This line shouldn't be changed. 
It is a puzzle how I let this line changed.